docs/CHANGES.md

```diff
@@ -61,6 +61,7 @@
   an instant room is created. [jcbrand]
 - Ensure consistent behavior from `show_controlbox_by_default` [jcbrand]
 - #365 Show join/leave messages for chat rooms.
+- #585 Fixes the duplication bug due to case sensivity in adding contacts.
```

Comment:
You've spilt up a single changelog entry and put your new entry in the middle of it.